nessie/src/test/java/org/apache/iceberg/nessie/TestMultipleClients.java

```diff
@@ -71,6 +71,18 @@ public void afterEach() throws Exception {
     anotherCatalog.close();
   }
 
+  @Test
+  public void testCreateNamespacesWithLevels() {
```

Comment:
Not sure how this test uses multiple clients (`TestMultipleClients`) and the difference to the test in `TestNessieIcebergClient`.